notebooks/Tutorial.ipynb

```diff
@@ -396,26 +396,26 @@
    "id": "8c7c6d3f",
    "metadata": {},
    "source": [
-    "### Spatial Alignment"
+    "### Homogenize Maps"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "df6070e8",
    "metadata": {},
    "source": [
-    "Spatial alignment by default aligns to the benchmark map, however one can also align to the candidate map:"
+    "Homegnization by default spatially aligns to the benchmark map, however one can also spatially align to the candidate map:"
```

Comment:
I'm suggesting alternate text in order to 1) introduce the homogenize accessor method more broadly and 2) clean up the language a bit:

`Homogenization is intended to help prepare two disparate maps for comparison. Currently, homogenization handles three sets of functionality:

1) *Spatial alignment:* matching the CRS's and coordinates of candidate and benchmark xarray maps. By default, the benchmark map is used as the target of this alignment but the candidate map can also be selected.
2) *Data type alignment:* in order to avoid precision warnings in the comparisons, dtypes are set to the highest precision dtype of the two maps.
3) *Data format conversion:* a vector data format benchmark map as a Geopanda's DataFrame can be passed which will be converted to the same xarray object as the candidate map with the same CRS and coordinates.`

We can leave any reference to the last point out until that's thrown in a notebook somewhere or just have it in for now.

Reply:
Notebook changed